src/components/notifications/GtxMessaging.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+    <div class="mb-3">
+        <label for="gtxmessaging-api-key" class="form-label">{{ $t("gtxMessagingApiKey") }}</label>
+        <HiddenInput id="gtxmessaging-api-key" v-model="$parent.notification.gtxMessagingApiKey" :required="true"></HiddenInput>
+    </div>
+    <div class="mb-3">
+        <label for="gtxmessaging-from" class="form-label">{{ $t("gtxMessagingFrom") }}</label>
+        <input id="gtxmessaging-from" v-model="$parent.notification.gtxMessagingFrom" type="text" class="form-control" required>
+    </div>
+    <div class="mb-3">
+        <label for="gtxmessaging-to" class="form-label">{{ $t("gtxMessagingTo") }}</label>
+        <input id="gtxmessaging-to" v-model="$parent.notification.gtxMessagingTo" type="text" pattern="^[\d-]{10,31}(@[\w\.]{1,})?$" class="form-control" required>
```

Comment:
the regex does not match what is documented in their docs

Reply:
sorry, updated regex